faster_whisper/audio.py

```diff
@@ -15,6 +15,27 @@
 import av
 import numpy as np
 
+# Audio Hyperparameters
```

Comment:
Should remove unused Hyperparameters (not needed to init and import  `TIME_PRECISION`)

Reply:
Done